Gemfile

```diff
@@ -86,6 +86,7 @@ gem 'googleauth'
 gem 'google-protobuf' # For Datadog Profiling
 gem 'govdelivery-tms', git: 'https://github.com/department-of-veterans-affairs/govdelivery-tms-ruby.git', tag: 'v4.0.0', require: 'govdelivery/tms/mail/delivery_method'
 gem 'gyoku'
+gem 'hexapdf'
```

Comment:
I believe this also needs the hexapdf package added to the Dockerfile?

Reply:
Thanks for reviewing - is there any documentation for how to update the Dockerfile? This is my first time adding a new gem to vets-api so any guidance here would be very helpful.